src/app/about/page.js

```diff
@@ -2,7 +2,7 @@
 
 import Image from "next/image";
 import mockup from "../images/mockup.png";
-import demo from "../images/demo.png";
+import demo2 from "../images/demo2.png";
```

Comment:
Can you rename this image to be "devdogswelcome.png" and the associated import name ("import devdogswelcome from "../images/devdogswelcome.png")?

Reply:
I made the appropriate changes